libs/components/ag-grid/src/lib/modules/ag-grid/header/header.component.html

```diff
@@ -35,12 +35,12 @@
         /></span>
       }
       @if (params?.enableSorting) {
-        <button
-          class="ag-sort-indicator-container sky-btn sky-btn-icon-borderless"
-          type="button"
-          (click)="onSortRequested($event)"
-        >
-          @if (sortOrder$ | async; as sortDirection) {
+        @if (sortOrder$ | async; as sortDirection) {
```

Comment:
This does technically change things to where it is not possible to have the "sortIndexDisplay` visible without `sortDirection`. However, based on my research of AG Grid - this isn't possible anyway as there is only ever an index when a column is sorting and part of a multi-sort scenario. So there is no functional change here.

We needed the screen reader text pulled out of the button so that it could be in the accessible display without the button also being rendered. We had an existing `:empty` css selector for this button; however, it wasn't working due to the addition of the screen reader text.

Reply:
Seems like you could still write a unit test for this behavior? On init, the sort button should not exist, and after clicking the heading it should?

Also, if the `:empty` style isn't being used anymore, let's remove it.

Reply:
I've added unit testing and removed the `:empty` style